src/views/PhoneSignIn.vue

```diff
@@ -85,6 +88,35 @@ export default {
         });
     },
 
+    saveLocalUserConfig() {

```

Comment:
So this component is now doing two things:

1. Signing in with the phone
2. Saving user config, setting Locale etc

I would suggest separating the logic (`PhoneSignIn.vue`, `SetUserProperties.vue`), so that when we implement email sign in/other auth protocols, you just need to implement the sign in logic.

I see some code for experiments, etc, which should also be out of this file.

Reply:
The code for experiments is just to redirect the user to the page where they came from. Nothing else.

Reply:
Looking at the separation of the logic now.

Reply:
Done